setup.py

```diff
@@ -11,7 +11,13 @@
     ]
 )
 
-install_requires = ["netCDF4", "threedi-api-client", "threedigrid", "pandas"]
+install_requires = [
+    "netCDF4",
+    "threedi-api-client",
+    "threedigrid",
+    "pandas",
+    "requests",
```

Comment:
Het netste is om het gelijk even alfabetisch te sorteren.
Klinkt misschien overdreven, maar het helpt wel: elk jaar ontdek ik op die manier wel eens een dubbeling in het lijstje (dat pas opvalt bij sorteren)!